packages/voice/src/VoiceConnection.ts

```diff
@@ -161,6 +162,32 @@ export type VoiceConnectionState =
 	| VoiceConnectionReadyState
 	| VoiceConnectionDestroyedState;
 
+export interface VoiceConnection extends EventEmitter {
```

Comment:
Why not use [tiny-typed-emitter](https://www.npmjs.com/package/tiny-typed-emitter) instead? Would make every method strictly typed then 👀

Reply:
@iCrawl removed this with https://github.com/discordjs/discord.js/commit/d4b41dd0815b493b599d4f4d1b6dd18cd99f91ea. My description mentioned it.